all_futures.gemspec

```diff
@@ -19,6 +19,9 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency "kredis", "~> 1.0"
   gem.add_dependency "activeentity", "~> 6.1"
+  gem.add_dependency "activemodel", ">= 4.0"
+  gem.add_dependency "activesupport", ">= 4.0"
```

Comment:
ActiveEntity already depends on AM and AS:

```rb
  s.add_dependency "activesupport", "~> 6.0", "< 7.0"
  s.add_dependency "activemodel", "~> 6.0", "< 7.0"
```
Doesn't that make this redundant?

Reply:
Well what if AE's dependency graph ever changes? Or you decide to roll your own?

I'd get the point if you were trying to stay Rails-agnostic, but I don't feel that's the direction here?